Submodule crt/aws-crt-cpp contains modified content
Submodule crt/aws-c-common contains modified content
diff --git a/crt/aws-crt-cpp/crt/aws-c-common/source/json.c b/crt/aws-crt-cpp/crt/aws-c-common/source/json.c
index 2f1630e..f0eb014 100644
--- a/crt/aws-crt-cpp/crt/aws-c-common/source/json.c
+++ b/crt/aws-crt-cpp/crt/aws-c-common/source/json.c
@@ -385,6 +385,7 @@ void aws_json_module_init(struct aws_allocator *allocator) {
 void aws_json_module_cleanup(void) {
     if (s_aws_json_module_initialized) {
         s_aws_json_module_allocator = NULL;
+        cJSON_InitHooks(NULL); // custom change to keep
         s_aws_json_module_initialized = false;
     }
 }
